src/Controls/samples/Controls.Sample.iOS/Maui.Controls.Sample.iOS-net6.csproj

```diff
@@ -1,4 +1,4 @@
-<Project Sdk="Microsoft.NET.Sdk">
+<Project Sdk="Microsoft.NET.Sdk.Razor">
```

Comment:
why this change?

Reply:
oh i understand now..

Reply:
Yes, the `.Razor` is what brings in BlazorWebView.

I would rather make `UseBlazor=true` bring this in, but I don't see a way to do this from a workload.

A workload doesn't have an `Sdk.props` file, which is what we'd need to import the `Razor` Sdk ourselves.